### ISSUE TITLE
Cannot install RStudio on Windows if a different user has it running on same system

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#12924]
### System details

    RStudio Edition : Desktop
    RStudio Version : 2023.03.0-386 (also 2022.12.x)
    OS Version      : Windows
    R Version       : any

### Steps to reproduce the problem

- in User Account "A" install RStudio normally then start it
- switch to a different user "B" (leave "A" logged in with RStudio running)
- try to install RStudio as user "B"

### Describe the problem in detail

- get an error from setup reading "RStudio is currently running. Please close it before installing a new version."
- setup checks for "rstudio.exe" process and if it sees one it won't continue
- this happens here: https://github.com/rstudio/rstudio/blob/b2d7353fdf45be8168d19db2de3dea9b6f84bc40/package/win32/cmake/modules/NSIS.template.in.electron#L994-L999

### Describe the behavior you expected

- in a multi-user Windows Server situation, a user should be able to install their own copy of RStudio even if other users already have it running
- so either a way to avoid this check (perhaps a command-line switch on the setup program), or at least have the check only look for rstudio.exe process owned by current user

That said, if we enable this, and a user tries to update over the version that is in use, other problems may occur. Perhaps the answer would be part of supporting non-admin installs as discussed here: [Allow RStudio installer to install without admin privilges #1634](https://github.com/rstudio/rstudio/issues/1634)

- [x] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [x] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [x] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [x] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by kevinushey** [comment]
https://nsis.sourceforge.io/Check_whether_your_application_is_running#Using_the_name_of_the_process_(with_Windows'_tasklist_command) could possibly be a way forward?

**Comment by kaijagahm** [comment]
"if you are using the RStudio installer" Does that mean there's a way to download RStudio without using the installer? I'd be interested in looking into that.

**Comment by gtritchie** [comment]
> Does that mean there's a way to download RStudio without using the installer? 

Yes! We also provide it as a zip file. You can just extract it and run rstudio.exe directly (or create a shortcut, etc.).

Go to the download page (https://posit.co/download/rstudio-desktop/), scroll down, and you'll see this option:

<img width="1189" alt="windows-zip" src="https://github.com/user-attachments/assets/3a0c5c21-55ba-49fa-9bb0-efec8a7e8f07">
